docs/network_disruption/prio.md

```diff
@@ -152,6 +152,43 @@ We first filter for all packets related to `gateway IP` and `node IP` and send t
 
 Finally, we apply a filter to enqueue all packets to class `1:4` whenever the `destination IP` is encompassed by the `hosts` field (see [this documentation](../../docs/network_disruption/hosts-and-services.md) for more details). In this case, a filter is applied for `10.0.1.254/32` and another for `10.0.1.255/32`. If no hosts were specified, a single filter is applied for `0.0.0.0/0` and no traffic is ends up in class `2:1`.
 
+### Network Disruption implementation for pod level with eBPF filters
+
+Tc can use eBPF filters in order to intercept packets at the application layer and allow the access to the payload of packets.
```

Comment:
Are there requirements for the host we're disrupting on for these ebpf filters to work? I don't see documentation about that (unless it's in the images?)

Reply:
Good point 👍  The host needs to have a Linux kernel with eBPF options.